waku/v2/protocol/waku_enr.nim

```diff
@@ -147,11 +180,16 @@ func toMultiAddresses*(multiaddrsField: seq[byte]): seq[MultiAddress] =
 
   return multiaddrs
 
+
+## ENR
+
 func init*(T: type enr.Record,
+           seqNum: uint64,
```

Comment:
Do we plan to implement some heuristic around the `seq` number at some point?

Reply:
I am exposing it since the discv5 protocol uses it to exchange records and update the stored ENR. If the number is stuck in `1`, the querying node stored records may not be updated.

IMO, we should develop the ENR sequence number heuristic after working on an integrated discovery strategy.